uint16[] input_voltage
uint16[] motor_hall_sensor
uint16[] overheating
uint16[] motor_encoder
uint16[] electronical_shock
uint16[] overload